third_party/move/move-model/src/ty.rs

```diff
@@ -459,6 +459,7 @@ impl Constraint {
         subs: &mut Substitution,
         loc: &Loc,
         other: &Constraint,
+        ctx_opt: Option<ConstraintContext>,
```

Comment:
Please add brief info to the function doc string on how this additional parameter is only used for additional error info.

Reply:
done